packages/testing/src/utils.ts

```diff
@@ -18,6 +18,8 @@ export async function waitOnNamespace(
       if (
         err.details.includes('workflow history not found') ||
         err.details.includes('Workflow executionsRow not found') ||
+        err.details.includes('operation GetCurrentExecution') ||
+        err.details.includes('operation GetWorkflowExecution encountered not found') ||
```

Comment:
There's no need to wait on namespaces anymore since 1.20 but it's better to improve the existing helper than to assume we're working with a new enough server for the time being.